tests/Rector/v13/v0/MigrateAddPageTSConfigToPageTsConfigFileRector/Fixture/extension3/ext_localconf.php.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Ssch\TYPO3Rector\Tests\Rector\v13\v0\MigrateAddPageTSConfigToPageTsConfigFileRector\Fixture;
+
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+$variable = 'ext_key';
+
+\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig(
+    '@import "EXT:' . $variable . '/Configuration/TSconfig/*/*.tsconfig"'
```

Comment:
could we do something about this line because i see many people do it this way and then the code won't be migrated which sucks.

Reply:
I have tried it with a naive approach. I guess there are edge case where people doing crazy things. But let´s give it a try and see what will be reported.